docker-compose.yml

```diff
@@ -118,6 +116,11 @@ services:
     image: ${VSI_COMMON_DOCKER_REPO}:os_${VSI_COMMON_TEST_OS_TAG_NAME-debian_8}
     volumes:
       - <<: *vsi_common_volume
+      # Since some of the integration tests bind test dirs in, make sure the
+      # tmp it is using is the same tmp dir
```

Comment:
```suggestion
      # Because some of the integration tests bind mount test dirs into the
      # container, ensure the tmp dir it is using is the same tmp dir
```